pyte/screens.py

```diff
@@ -186,40 +186,27 @@ def __repr__(self):
     def display(self):
         """A :func:`list` of screen lines as unicode strings."""
         def render(line):
-            it = iter(line)
-            while True:
-                char = next(it).data
+            wide_char = False
```

Comment:
Is there a reason for rewriting this using a flag?

Also, could you include the `in` optimization?

```python
if col not in line:
    yield " "  # or default_char.data
```